interactive_engine/executor/ir/clients/rust/client/src/physical_builder.rs

```diff
@@ -209,6 +209,15 @@ impl Plan {
         self
     }
 
+    // Notice that this is used to set the meta_data of the **Last Appended OP**
+    pub fn with_meta_data(&mut self, meta_data: Option<Vec<pb::physical_opr::MetaData>>) {
+        if let Some(op) = self.plan.last_mut() {
+            if let Some(meta_data) = meta_data {
+                op.op_meta = meta_data;
```

Comment:
Probably `meta_data` is a better name than `op_meta`.

Reply:
Done.